translations/catroweb.en.yml

```diff
@@ -173,10 +173,18 @@ programs:
   deleted_popup: The comment was successfully deleted.
   deleted_popup_header: Deleted
   noAdminRights: You have no admin rights. Please Log in as an Admin.
+  description: description
+  notesAndCredits: notes and credits
```

Comment:
there are already strings defined for this, even though the first letter is capitalized, it's fine just to use that

Reply:
Yeah, I saw those. I wasn't sure if we wanted to use them so I created new ones, but they are now deleted and it's using the existing ones